client/transaction-pool/Cargo.toml

```diff
@@ -21,23 +21,24 @@ intervalier = "0.4.0"
 log = "0.4.8"
 parity-util-mem = { version = "0.7.0", default-features = false, features = ["primitive-types"] }
 parking_lot = "0.10.0"
-prometheus-endpoint = { package = "substrate-prometheus-endpoint", path = "../../utils/prometheus", version = "0.8.0"}
-sc-client-api = { version = "2.0.0", path = "../api" }
-sc-transaction-graph = { version = "2.0.0", path = "./graph" }
-sp-api = { version = "2.0.0", path = "../../primitives/api" }
-sp-core = { version = "2.0.0", path = "../../primitives/core" }
-sp-runtime = { version = "2.0.0", path = "../../primitives/runtime" }
-sp-tracing = { version = "2.0.0", path = "../../primitives/tracing" }
-sp-transaction-pool = { version = "2.0.0", path = "../../primitives/transaction-pool" }
-sp-blockchain = { version = "2.0.0", path = "../../primitives/blockchain" }
-sp-utils = { version = "2.0.0", path = "../../primitives/utils" }
+prometheus-endpoint = { package = "substrate-prometheus-endpoint", version = "0.8.0"}
+sc-client-api = { version = "2.0.0"}
+sc-transaction-graph = { version = "2.0.0"}
+sp-api = { version = "2.0.0"}
+sp-core = { version = "2.0.0"}
+sp-runtime = { version = "2.0.0"}
+sp-tracing = { version = "2.0.0"}
+sp-transaction-pool = { version = "2.0.0"}
+sp-blockchain = { version = "2.0.0"}
+sp-utils = { version = "2.0.0"}
```

Comment:
same as previously- why use substrate crates while having these locally?
If none of them are modified by us, we should remove them.

Reply:
`Cargo.toml` was taken from substrate repo  where all dependencies are aprt of repository above are not available in mangata repo